es-vue-base/src/lib-components/EsAccordion.vue

```diff
@@ -1,15 +1,22 @@
 <template>
-    <div class="EsAccordion border-bottom border-light rounded-bottom">
+    <div
+        class="EsAccordion border-bottom border-light"
+        :class="{
+            ['EsAccordion--' + accordionVariant]: true,
+            'rounded-bottom': accordionVariant === 'default',
+        }">
         <header
             role="tab"
             class="position-relative">
             <component
                 :is="headingTag"
-                class="EsAccordion-heading mb-0 align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 text-body font-size-100"
+                class="EsAccordion-heading mb-0 align-items-center d-flex font-weight-bold justify-content-between py-100 rounded-0 text-body text-body font-size-100 font-size-lg-300"
                 :class="{
-                    'bg-gray-200': isVisible,
-                    'bg-white': !isVisible,
+                    'bg-gray-200': isVisible && accordionVariant === 'default',
+                    'bg-white': !isVisible && accordionVariant === 'default',
                     'EsAccordion-heading--visible': isVisible,
+                    'px-100 px-sm-200': accordionVariant === 'default',
+                    'px-0': accordionVariant === 'minimal',
```

Comment:
to make the heading tag larger in minimal variant like in figma,
`'px-0 font-size-200 font-size-lg-400': accordionVariant === 'minimal',`